### ISSUE TITLE
Pupil Player & Capture are not work in MacOS Big Sur

### ISSUE THREAD

**Issue by ZIwen-Ruan** [pupil-labs/pupil#2079]
When I upgraded my MacOS into the Big Sur, I'm not able to open the Player or Capture application. And it seems that the Pupil Player or Capture even not appear in the system process manager.

**Comment by blufu1** [comment]
Same here. I am running a M1 MacBook with Big Sur and neither the Player nor Capture can be started.

**Comment by papr** [comment]
We can confirm that the bundled Pupil Core software currently does not work on Big Sur due to an incompatibility between Python and macOS Big Sur. See https://bugs.python.org/issue41100 for reference.

There was a release with fixes for the compatibility for Python 3.9.1 recently but it is considered experimental. Therefore, we will not be able to include it in our next update yet. Our first bundled release to support Big Sur will likely be released mid to end of January.

As an alternative, you can try the newly released 3.9.1 Python installer and run Pupil Core software from source. Please see the macOS dependency instructions here. Please be aware that these instructions have not been tested on macOS Big Sur yet.

If you decide to give running from source a chance, we would appreciate hearing from your experiences.

**Comment by papr** [comment]
Hi @ZIwen-Ruan @blufu1 

Thank you for your patience regarding Pupil Core software + macOS Big Sur compatibility. We have been able to resolve _most_ of the compatibility issues and would like to share a pre-release - Pupil Core v3.1.

**Download** the pre-release here: https://drive.google.com/file/d/1U9EB71BpX071xJZ1Fq2-jAj2R0dTFY9p/view?usp=sharing

**Explanation**: macOS 11 Big Sur introduced backwards-incompatible changes that caused earlier versions of Pupil Core (and other software) to not run properly. 

**Warning**: Not all features of the software run as expected on Apple Silicon M1 macs. We have disabled the following features for Apple M1 macs:
- Screen marker calibration choreography in *fullscreen* and *window* mode
- Single marker calibration choreography in *fullscreen* and *window* mode
- Single marker calibration choreography remains available in *physical* mode.

We hope to have an official release that provides full support for M1 silicon macs in the coming weeks. As soon as this is the case, we will let you know.

We look forward to your feedback.

Best,
Pablo
